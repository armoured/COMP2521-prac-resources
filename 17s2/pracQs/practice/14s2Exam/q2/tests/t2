seq 15 -1 1 | ./q2
