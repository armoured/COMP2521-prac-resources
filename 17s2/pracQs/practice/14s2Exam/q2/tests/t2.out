[ 0] 12
[ 1] 13->1
[ 2] 14->2
[ 3] 15->3
[ 4] 4
[ 5] 5
[ 6] 6
[ 7] 7
[ 8] 8
[ 9] 9
[10] 10
[11] 11
